web/Export/PhpCsExporter.ts

```diff
@@ -73,7 +73,6 @@ export default class PhpCsExporter implements ExporterInterface {
         }
         lines.push(INDENT + '])');
         lines.push(INDENT + '->setFinder(PhpCsFixer\\Finder::create()');
-        lines.push(INDENT + INDENT + '->exclude(\'vendor\')');
         lines.push(INDENT + INDENT + '->in(__DIR__)');
```

Comment:
maybe a good middle ground?

```suggestion
        lines.push(INDENT + INDENT + '// ->exclude(\'folder-to-exclude\') // if you want to exclude some folders, you can do it like this!');
        lines.push(INDENT + INDENT + '->in(__DIR__)');
```

Reply:
Yep,, I love it!